zipkin-lens/src/types/AppThunk.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2020 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { Action } from 'redux';
+import { ThunkAction } from 'redux-thunk';
+
+import RootState from './RootState';
+
+type AppThunk<ReturnType = void> = ThunkAction<
```

Comment:
Instead of aliasing types, it's common to override them instead in `types.d.ts`

Reply:
It is not overriding.
`AppThunk` is an application specific type which is defined to avoid repeating the same type declaration for every thunk function.

https://redux-toolkit.js.org/tutorials/advanced-tutorial#adding-a-reusable-thunk-function-type